### DISCUSSION TITLE
Tag / list releases properly

### DISCUSSION THREAD

**Discussion by mfts** [resend/react-email#1084] [Ideas]
I like to stay up to date with new package version. But when I don't see what the changes to a previous version are it's super difficult to know if the minor version is gonna introduce / deprecate a prop / component that I'm using and therefore break my app. 

The npm pages links to this repo, but here are no tags or releases, so I'm digging through last commits based on timestamps.
https://www.npmjs.com/package/react-email

Thanks
Marc

**Comment by gabrielmfern** [comment]
Not so sure if this is an issue, can you maybe elaborate a bit more?
Also, you can find a change-log that we are trying to keep up to date on our docs [here](https://react.email/docs/changelog).

**Comment by mfts** [comment]
It's not a bug. You can convert it into a discussion. 

Usually when a new minor/major version is released I check versions or tags on GitHub to see what changed in the code. 

Many oss packages release like that.

- https://github.com/vercel/ai/releases
- https://github.com/4lejandrito/next-plausible/tags

I'm not asking for beautiful release notes, just tag software when you are shipping a release :)

**Comment by zenorocha** [comment]
This is super relevant. We should definitely do that.

**Comment by gabrielmfern** [comment]
@mfts Would love to tell you that our latest canary release is already tagged, and we will be tagging them from now on. 

Probably for the next releases we make we will also be adding it to GitHub based on the tags.

**Reply by mfts** [reply]
Awesome @gabrielmfern 😎 thanks for sharing the great news
